bukuserver/templates/bukuserver/home.html

```diff
@@ -4,6 +4,7 @@
 {% block head %}
   {{ super() }}
   {{ buku.close_if_popup() }}
+  {{ buku.focus('form[action="/"]') }}
```

Comment:
When loading the main page, focus should probably be on the search form (so that the user can immediately start typing)